{"uuid": "956ea669-4867-4ff5-b5fe-b772eab70c18", "children": ["029223a0-9c42-45a8-a7a9-fbc64bc5662c", "1c02a878-5d9b-4fd2-9fc5-10f16237cbbe"], "befores": [{"name": "setup_profiles", "status": "broken", "statusDetails": {"message": "KeyError: 'ifname'\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/pluggy/callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/wlan-testing/tests/e2e/basic/conftest.py\", line 79, in setup_profiles\n    return_var = fixtures_ver.setup_profiles(request, param, setup_controller, testbed, get_equipment_id,\n  File \"/wlan-testing/tests/fixtures_2x.py\", line 292, in setup_profiles\n    ap_wifi_data = ap_ssh.get_interface_details()\n  File \"/wlan-testing/tests/../libs/apnos/apnos.py\", line 403, in get_interface_details\n    wifi_info[j[\"ifname\"]] = [j[\"config\"][\"ssid\"], j[\"config\"][\"encryption\"], \"\"]\n"}, "attachments": [{"name": "ucentral_config: ", "source": "bd1dc9bb-4510-41cc-9cb7-f8f6704d90fc-attachment.attach"}, {"name": "Time Took to apply Config: 55.996225118637085", "source": "29604f12-434d-4f39-a5af-1b969dce650f-attachment.attach"}, {"name": "Success : Active Config in AP: ", "source": "d6f1323a-e37e-48f1-8cf2-e9a42504b77b-attachment.attach"}, {"name": "AP LOgs: ", "source": "d68a9b5c-47da-43b1-94c5-60ac15d583ac-attachment.attach"}], "start": 1628902339723, "stop": 1628902424632}], "start": 1628902339723, "stop": 1628902425241}